drake/doc/developers.rst

```diff
@@ -8,32 +8,75 @@ The standard github workflow is to fork the drake repository into your
 own github account, push your changes into a branch on that account,
 then (when your code is definitely ready) open a `pull request
 <https://help.github.com/articles/using-pull-requests/>`_ via the
-github website.  Please see the links below for information on code
-style, etc, and be prepared to engage in active code review on your
-pull request.
-
-Your code will undergo a code review and will run the unit and
-regression tests on a number of build servers.  You can run those
-tests locally by running ``make test`` from the command line -- note
-that there are a lot of computationally demanding tests and this could
-run for a few hours depending on your machine.
-
-Your change must include unit tests that protect it against regressions,
-and those tests must pass on all platforms supported by Drake.  Please
-use the googletest framework, which is already available in the superbuild.
```

Comment:
c++ code should use googletest.  in matlab, every example and static class method runs as a unit test and fails if it times out or calls matlab's `error` method.  you can direct users to read the doc at `drake/util/unitTest.m` for details on `%NOTEST` and `%TIMEOUT`, etc.